include/coap2/coap_asn1_internal.h

```diff
@@ -56,7 +56,7 @@ size_t asn1_len(const uint8_t **ptr);
  * @param constructed  1 if current tag is constructed
  * @param class  The current class of the tag
  *
- * @return The tag value.@p ptr is udated to pass the tag.
+ * @return The tag value.@p ptr is updated to pass the tag.
```

Comment:
`pass` also needs updating to `past`, or perhaps a better phrase is `be after`.
````
* @return The tag value.@p ptr is updated to be after the tag.
````
in both places.

The build failure is due to a retrieving lwip code failure.

Reply:
thx @mrdeep1